AWSSDKSwiftCLI/Sources/AWSSDKSwiftCLI/Models/ReleaseNotesBuilder.swift

```diff
@@ -15,24 +15,73 @@ struct ReleaseNotesBuilder {
     let repoOrg: PrepareRelease.Org
     let repoType: PrepareRelease.Repo
     let commits: [String]
+    var isTest: Bool = false
     
     // MARK: - Build
     
-    func build() -> String {
-        let contents = [
-            "## What's Changed",
-            buildCommits(),
-            .newline,
-            "**Full Changelog**: https://github.com/\(repoOrg.rawValue)/\(repoType.rawValue)/compare/\(previousVersion)...\(newVersion)"
+    func build() throws -> String {
+        let baseContent: [String] = (
+            ["## What's Changed"] +
+            buildSDKChangeSection()
```

Comment:
Should we maybe put SDK changes below service-related changes?

Reply:
Sure, that might make more sense since most customers would just care about service client changes. Changed as suggested.